english/hospital_bed/hospital_bed.md

```diff
@@ -0,0 +1,8 @@
+Just then, a crazed ostrich smacks into you at full-speed.  Your phone flies out of your hands and you hit the pavement.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.